<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consulta</class>
 <widget class="QDialog" name="Consulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta Reparacion</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="RepID">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TrabajoConsulta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Equipo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cant</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Factura</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comentarios</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RepID</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lAgente">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Agente</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CambioFrep">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CambioFrep</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Consulta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Consulta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
